src/settings.example.py

```diff
@@ -417,10 +417,13 @@
             'description': 'Waage 1',
             'type': 'HX711',
             'enabled': False,
+            'dualchannel': False,    # this will activate reading channel B
             'pin_dout': 'P22',
             'pin_pdsck': 'P21',
-            'offset': -73000,
-            'scale': 4.424242,
+            'offset': -73000,   # channel A
+            'scale': 4.424242,  # channel A
+            'scaleB': 10767,    # channel B
+            'offsetB': 11642,   # channel B            
```

Comment:
Can you swap both lines that they're in the same order like `offset` and `scale`?

Reply:
Done. Not sure how that happened...

Reply:
Thanks!